frontend/views/pages/GroupChat.vue

```diff
@@ -75,12 +75,14 @@ page(pageTestName='groupChat' :miniHeader='isGroupDirectMessage()')
         i18n(:args='{ messagesCount: pinnedMessages.length }') {messagesCount} Pinned
       template(v-if='!isGroupDirectMessage()')
         span(v-if='pinnedMessages.length') ∙
-        i18n.is-unstyled.c-link(
-          tag='button'
+        button.is-unstyled.c-link(
+          type='button'
           @click='openModal("ChatMembersAllModal")'
-          :args='{ numMembers: summary.numberOfMembers  }'
           data-test='channelMembers'
-        ) {numMembers} members
+        )
+          i18n(v-if='summary.numberOfMembers === 1' :args='{ num: summary.numberOfMembers }') {num} member
```

Comment:
In this case, there's no need for the `{num}` variable in the string or the `:args` :), you know it's `1`

Reply:
Right, it's a good point.